### DISCUSSION TITLE
Change FleatherToolbar theme color?

### DISCUSSION THREAD

**Discussion by OlaveraLuffy** [fleather-editor/fleather#195] [Q&A] [answered]
Hi! I would like to change the color of the icons in the fleathertoolbar. I was wondering how can I do this? As you can see in the image, I cannot see the icons in the toolbar if such icons is not toggled. Appreciate if someone would help. Thanks!

![1](https://github.com/fleather-editor/fleather/assets/38834838/4b6bceca-c446-45d2-8669-862a79a618e4)

**Comment by Amir-P** [comment, marked as answer]
If you're using basic toolbar, you can change the colors by changing `primaryIconTheme`, `iconTheme` and `disabledColor` in `ThemeData`. Use this as an example:

```dart
Theme(
  data: Theme.of(context).copyWith(
      disabledColor: Colors.red, // used for disabled icons
      primaryIconTheme: IconThemeData(color: Colors.green), // used for toggled icons
      iconTheme: IconThemeData(color: Colors.blue)), // used for not toggled icons
  child: FleatherToolbar.basic(controller: _controller!),
),
```

**Reply by OlaveraLuffy** [reply]
Thank you for your help!

**Reply by Amir-P** [reply]
You're welcome!
